Cargo.toml

```diff
@@ -359,7 +362,7 @@ required-features = ["sqlite"]
 [[test]]
 name = "mysql"
 path = "tests/mysql/mysql.rs"
-required-features = ["mysql"]
+required-features = ["mysql", "compression"]
```

Comment:
Instead of requiring the `compression` feature, you should either just `#[cfg]` out those tests or create a new test target.

Reply:
I have added targets to the pipeline in order to run existing tests with enabled compression. Thus this change is not needed.